x-pack/platform/plugins/shared/onechat/server/routes/index.ts

```diff
@@ -7,7 +7,9 @@
 
 import type { RouteDependencies } from './types';
 import { registerToolsRoutes } from './tools';
+import { registerMCPRoutes } from './mcp';
 
 export const registerRoutes = (dependencies: RouteDependencies) => {
   registerToolsRoutes(dependencies);
+  registerMCPRoutes(dependencies);
```

Comment:
is this whole plugin behind a feature flag? If not, I suggest putting the route registration behind a feature flag. There is aparantly some upcoming guidance to be less cavalier about publishing APIs and then breaking them later, even if they're "tech preview." So until we're totally confident in the API design, we probably want to ensure they aren't exposed.